[
   {
      "body" : "You can try rerun by cleaning app data/bitcoin except wallet.dat \r\nI think it will help , please keep wallet.dat safe",
      "created_at" : "2016-11-01T15:18:09Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9057#issuecomment-257594235",
      "id" : 257594235,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9057",
      "updated_at" : "2016-11-01T15:18:09Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/257594235",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/12095370?v=3",
         "events_url" : "https://api.github.com/users/joshafest/events{/privacy}",
         "followers_url" : "https://api.github.com/users/joshafest/followers",
         "following_url" : "https://api.github.com/users/joshafest/following{/other_user}",
         "gists_url" : "https://api.github.com/users/joshafest/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/joshafest",
         "id" : 12095370,
         "login" : "joshafest",
         "organizations_url" : "https://api.github.com/users/joshafest/orgs",
         "received_events_url" : "https://api.github.com/users/joshafest/received_events",
         "repos_url" : "https://api.github.com/users/joshafest/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/joshafest/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/joshafest/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/joshafest"
      }
   },
   {
      "body" : "Was the previous run of bitcoin-core shut down uncleanly? (E.g. crashed for some other reason, power went out, terminated from Task Manager, etc.) If you're not sure, we might be able to tell that if you posted the full debug.log.",
      "created_at" : "2016-11-01T16:29:55Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9057#issuecomment-257615083",
      "id" : 257615083,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9057",
      "updated_at" : "2016-11-01T16:29:55Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/257615083",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/1047859?v=3",
         "events_url" : "https://api.github.com/users/kazcw/events{/privacy}",
         "followers_url" : "https://api.github.com/users/kazcw/followers",
         "following_url" : "https://api.github.com/users/kazcw/following{/other_user}",
         "gists_url" : "https://api.github.com/users/kazcw/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/kazcw",
         "id" : 1047859,
         "login" : "kazcw",
         "organizations_url" : "https://api.github.com/users/kazcw/orgs",
         "received_events_url" : "https://api.github.com/users/kazcw/received_events",
         "repos_url" : "https://api.github.com/users/kazcw/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/kazcw/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/kazcw/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/kazcw"
      }
   },
   {
      "body" : "\nThanks, but I have done that many times. I suspect it can be my Norton Firewall doing the problems.\n\nBest regards,\nHans N .LÃÂ¸vbrÃÂ¦k\n\nTirsdag 1. November 2016 16:18 CET skrev joshafest <notifications@github.com>:\nÃÂ \nYou can try rerun by cleaning app data/bitcoin except wallet.dat\nI think it will help , please keep wallet.dat safe\nÃ¢ÂÂ\nYou are receiving this because you authored the thread.\nReply to this email directly, view it on GitHub, or mute the thread.ÃÂ \n\n\nÃÂ \n",
      "created_at" : "2016-11-01T22:23:47Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9057#issuecomment-257717249",
      "id" : 257717249,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9057",
      "updated_at" : "2016-11-01T22:23:47Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/257717249",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/16051690?v=3",
         "events_url" : "https://api.github.com/users/Hanslo/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Hanslo/followers",
         "following_url" : "https://api.github.com/users/Hanslo/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Hanslo/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Hanslo",
         "id" : 16051690,
         "login" : "Hanslo",
         "organizations_url" : "https://api.github.com/users/Hanslo/orgs",
         "received_events_url" : "https://api.github.com/users/Hanslo/received_events",
         "repos_url" : "https://api.github.com/users/Hanslo/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Hanslo/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Hanslo/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Hanslo"
      }
   },
   {
      "body" : "\nI suspect now it's maybe my Norton Firewall doing the problems\n\nBest regards,\nHans N. LÃÂ¸vbrÃÂ¦k\n\nTirsdag 1. November 2016 17:30 CET skrev kazcw <notifications@github.com>:\nÃÂ \nWas the previous run of bitcoin-core shut down uncleanly? (E.g. crashed for some other reason, power went out, terminated from Task Manager, etc.) If you're not sure, we might be able to tell that if you posted the full debug.log.\nÃ¢ÂÂ\nYou are receiving this because you authored the thread.\nReply to this email directly, view it on GitHub, or mute the thread.ÃÂ \n\n\nÃÂ \n",
      "created_at" : "2016-11-01T22:25:28Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9057#issuecomment-257717611",
      "id" : 257717611,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9057",
      "updated_at" : "2016-11-01T22:25:28Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/257717611",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/16051690?v=3",
         "events_url" : "https://api.github.com/users/Hanslo/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Hanslo/followers",
         "following_url" : "https://api.github.com/users/Hanslo/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Hanslo/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Hanslo",
         "id" : 16051690,
         "login" : "Hanslo",
         "organizations_url" : "https://api.github.com/users/Hanslo/orgs",
         "received_events_url" : "https://api.github.com/users/Hanslo/received_events",
         "repos_url" : "https://api.github.com/users/Hanslo/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Hanslo/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Hanslo/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Hanslo"
      }
   },
   {
      "body" : "My Bitcoin Core no more crash, but when I copied my backup wallet inside it stopped to sync.",
      "created_at" : "2016-11-02T04:26:19Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9057#issuecomment-257769424",
      "id" : 257769424,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9057",
      "updated_at" : "2016-11-02T04:26:19Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/257769424",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/16051690?v=3",
         "events_url" : "https://api.github.com/users/Hanslo/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Hanslo/followers",
         "following_url" : "https://api.github.com/users/Hanslo/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Hanslo/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Hanslo",
         "id" : 16051690,
         "login" : "Hanslo",
         "organizations_url" : "https://api.github.com/users/Hanslo/orgs",
         "received_events_url" : "https://api.github.com/users/Hanslo/received_events",
         "repos_url" : "https://api.github.com/users/Hanslo/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Hanslo/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Hanslo/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Hanslo"
      }
   },
   {
      "body" : "Closing this as the original issue is fixed. @Hanslo If there is a second different problem, can you please open a new Issue, with more information.",
      "created_at" : "2016-11-11T11:14:12Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9057#issuecomment-259936100",
      "id" : 259936100,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9057",
      "updated_at" : "2016-11-11T11:14:12Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/259936100",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/863730?v=3",
         "events_url" : "https://api.github.com/users/fanquake/events{/privacy}",
         "followers_url" : "https://api.github.com/users/fanquake/followers",
         "following_url" : "https://api.github.com/users/fanquake/following{/other_user}",
         "gists_url" : "https://api.github.com/users/fanquake/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/fanquake",
         "id" : 863730,
         "login" : "fanquake",
         "organizations_url" : "https://api.github.com/users/fanquake/orgs",
         "received_events_url" : "https://api.github.com/users/fanquake/received_events",
         "repos_url" : "https://api.github.com/users/fanquake/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/fanquake/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/fanquake/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/fanquake"
      }
   }
]
